### ISSUE TITLE
Error email from Google. (split of zero)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cberetta/gmail-delay-send#78]
```
Summary:
Error message   TypeError: method "split" of zero can not be called (line 68, 
File "Utils, project "GmailDelaySendWeb")
Number          144
```

Original issue reported on code.google.com by `blairk...@gmail.com` on 25 Jun 2013 at 7:36

**Comment by GoogleCodeExporter** [comment]
```
I think an empty email draft should reproduce this issue.
```

Original comment by `blairk...@gmail.com` on 25 Jun 2013 at 7:40

**Comment by GoogleCodeExporter** [comment]
```
Fixed in b7afd980fb2f

Will include in next rollout.
```

Original comment by `blairk...@gmail.com` on 25 Jun 2013 at 10:45

**Comment by GoogleCodeExporter** [comment]
```
Fixed in change: 
https://code.google.com/p/gmail-delay-send/source/detail?r=b7afd980fb2f6beb08e62
7e67f3d8c84e1051342
```

Original comment by `blairk...@gmail.com` on 1 Jul 2013 at 8:59
- Changed state: **Fixed**
